### ISSUE TITLE
Server Crashes when Dynamite Lit near Bedrock

### ISSUE THREAD

**Issue by padmanek** [StrangeLoopGames/EcoIssues#25301]
It's does not always happen, but chances are around **10-25%.**

**Comment by D3nnis3n** [comment]
Please post the server log with the crash.

**Comment by padmanek** [comment]
This one was today on latest patch 11.1.4, we removed the dynamite near bedrock restriction law and went out to test. Crashed on 8th dynamite or so.
[log_241205120026.log](https://github.com/user-attachments/files/18030668/log_241205120026.log)

This one was on 28.11.24, right after hotfix 11.1.2 which claimed to fix the issue came out. We also removed the restriction law to test, crashed within 1 minute.
[log_241128120349.log](https://github.com/user-attachments/files/18030685/log_241128120349.log)

We have not had any other server crashed since this fresh wipe launched 2 weeks ago.
